pkg/v1/mutate/mutate.go

```diff
@@ -113,6 +113,29 @@ func Config(base v1.Image, cfg v1.Config) (v1.Image, error) {
 	return ConfigFile(base, cf)
 }
 
+// ManifestAnnotations mutates the provided v1.Image to have the provided annotations
+func ManifestAnnotations(base v1.Image, annotations map[string]string) (v1.Image, error) {
```

Comment:
I'd just call this `Annotations` because most of the other functions assume an image as well. If we want to extend this to other things, we can follow the `IndexMediaType` pattern and just do `IndexAnnotations`.

Ideally, this function would take a Describable, but then it's hard to know what to return :(

Reply:
done, thank you so much 🙏